packages/ui/src/proposals/components/ProposalDetails/ProposalDetails.tsx

```diff
@@ -78,6 +80,16 @@ export const ProposalDetails = ({ proposalDetails }: Props) => {
       ] as RenderNode[]
     }
 
+    if (proposalDetails?.type === 'fundingRequest') {
+      return [
+        {
+          renderType: 'Amount',
+          label: 'Current WG Budget',
+          value: group?.budget,
+        },
+      ] as RenderNode[]
+    }
+
```

Comment:
I don't think this works. I'm not sure why but the WG budget that this proposal should show is the content WG. 

Here `proposalDetails?.group?.id` doesn't exist so the budget remain at `0`.
Check https://dao-git-fork-vrrayz-funding-request-multiple-r-8b2aed-joystream.vercel.app/#/proposals/preview/374 (if the QN let you)
![Screen Shot 2023-08-31 at 11 26 12](https://github.com/Joystream/pioneer/assets/6571453/e5e1f039-a0be-4947-9b73-7c6eeeee6332)